features/AuthController.feature

```diff
@@ -56,6 +82,66 @@ Feature: Auth Controller
       | _id        | _source.userId | _source.ttl | _source.expiresAt | _source.description | _source.fingerprint |
       | "_STRING_" | "test-admin"   | -1          | -1                | "Sigfox API key"    | "_STRING_"          |
 
+  # auth:login =================================================================
+
+  @security @http
+  Scenario: Auth login with cookies
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
```

Comment:
```suggestion
      | body.username       | "test-admin"  |
      | body.password        | "password"  |
```

Reply:
Same as above, It seems that I cannot construct "object on the fly"